roles/core/display_tuning/molecule/default/molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:  |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
+platforms:
+  - name: instance
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-ubi8}-ansible:latest"
```

Comment:
Can you default to centos8 in all the molecule.yml files (like #162)?

Reply:
done.